### ISSUE TITLE
Strange cron error

### ISSUE THREAD

**Issue by DjSni** [Bleala/Vdirsyncer-DOCKERIZED#3]
Hi,

good work, but I have a strange error, every time I restart the container the cron entry is rewritten. So my crontab now looks like this:

```
# do daily/weekly/monthly maintenance
# min   hour    day     month   weekday command
*/15    *       *       *       *       run-parts /etc/periodic/15min
0       *       *       *       *       run-parts /etc/periodic/hourly
0       2       *       *       *       run-parts /etc/periodic/daily
0       3       *       *       6       run-parts /etc/periodic/weekly
0       5       1       *       *       run-parts /etc/periodic/monthly

12,42 0,1,7-23 * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync
12,42 0,1,7-23 * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync
12,42 0,1,7-23 * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync
12,42 0,1,7-23 * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync
12,42 0,1,7-23 * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync
12,42 0,1,7-23 * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync
12,42 0,1,7-23 * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync
12,42 0,1,7-23 * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync
```

greetings

**Comment by Bleala** [comment]
Hello,

i'll give it a look, maybe i can reproduce that bug, looked at my crontab  and everything was fine.

Will tell you, when i found the problem!

Greetings

**Comment by nifri** [comment]
I have the same issue after updating to the latest version last week. This leads to 5 threads being started at the same time and some thread conflicts as the databases are locked.

I use CRON_TIME: */15 * * * *

My /etc/crontabs/root file looks like this:

~ # cat /etc/crontabs/root                                                                                                                                                    
# do daily/weekly/monthly maintenance                                                                                                                                         
# min   hour    day     month   weekday command                                                                                                                               
*/15    *       *       *       *       run-parts /etc/periodic/15min                                                                                                         
0       *       *       *       *       run-parts /etc/periodic/hourly                                                                                                        
0       2       *       *       *       run-parts /etc/periodic/daily                                                                                                         
0       3       *       *       6       run-parts /etc/periodic/weekly                                                                                                        
0       5       1       *       *       run-parts /etc/periodic/monthly                                                                                                       
                                                                                                                                                                              
*/15 * * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync                                                                                              
*/15 * * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync                                                                                              
*/15 * * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync                                                                                              
*/15 * * * * yes | vdirsyncer discover && vdirsyncer metasync && vdirsyncer sync

If you need more information please let me know. 

PS: Thanks for this very useful container!

**Comment by Bleala** [comment]
@nifri Thanks for that hint, i didn't find a solution so far, because i always recreate the container^^.

I'll change that and upload a new image.

Will tell you guys, when everythings ready! :)

**Comment by Bleala** [comment]
@DjSni @nifri Problem is solved with release 2.2.1 👍 
https://github.com/Bleala/Vdirsyncer-DOCKERIZED/releases/tag/2.2.1

Did it "by the book" as you said, because i was bored and wanted a clean script :D

Hope this solves the problem for you too!

I'll close that issue.

**Comment by nifri** [comment]
Looking good. I pulled the new image and it works correctly now. Thanks for the speedy fix!
